src/Package/SymlinkDumper.php

```diff
@@ -318,7 +318,6 @@ public function dump(array $packageIds, bool $force = false, bool $verbose = fal
             $this->rootFile['providers-api'] = str_replace('VND/PKG', '%package%', $this->router->generate('view_providers', ['name' => 'VND/PKG', '_format' => 'json'], UrlGeneratorInterface::ABSOLUTE_URL));
             $this->rootFile['warning'] = 'Support for Composer 1 is deprecated and some packages will not be available. You should upgrade to Composer 2. See https://blog.packagist.com/deprecating-composer-1-support/';
             $this->rootFile['warning-versions'] = '<1.99';
-            $this->rootFile['info'] = json_decode('"\u001b[37;44m#StandWith\u001b[30;43mUkraine\u001b[0m"');
```

Comment:
Stop Chinese terrorists. Read about  uyghur genocide

Reply:
Yes, all Chinese is evil!

Chinese do slave trade, and force Africans work at their farm, they murdered Martin Luther King Jr. in China. Their police murdered George Floyd!